augur/dates/__init__.py

```diff
@@ -120,8 +120,6 @@ def get_numerical_date_from_value(value, fmt=None, min_max_year=None):
             ambig_date = AmbiguousDate(value, fmt=fmt).range(min_max_year=min_max_year)
         except InvalidDate as error:
             raise AugurError(str(error)) from error
-        if ambig_date is None or None in ambig_date:
-            return [None, None] #don't send to numeric_date or will be set to today
```

Comment:
_non-blocking_

> `AmbiguousDate().range()` will either return a tuple of (date, date) or raise an exception, so this condition will never be true.

This is the sort of thing type annotations and checking are good at catching is not possible and also checking that it stays impossible in the future.